pkg/mgrconfig/load.go

```diff
@@ -94,12 +94,44 @@ func defaultValues() *Config {
 		Procs:          6,
 		PreserveCorpus: true,
 		Experimental: Experimental{
-			RemoteCover: true,
-			CoverEdges:  true,
+			RemoteCover:      true,
+			CoverEdges:       true,
+			DescriptionsMode: DescriptionsMode(ManualDescriptions).String(),
```

Comment:
Change `DescriptionsMode(ManualDescriptions).String()` to `manualDescriptions`.

Reply:
Done!